File: programme/random-number/random-number.py
# Write a programme to generate a random number in range n1, n2

import random


def random_number(number1, number2):
    return random.randint(number1, number2)


range_start = 0
range_end = 100
print("Start range :", range_start)
print("End   range :", range_end)
print("\nRandom number :", random_number(range_start, range_end))

File: programme/random-number/random-number.c
// Write a programme to generate a random number in range n1-n2

/*
Programm take as given that inputs are always lower<upper
Normally we would need a input control to confirm that the uper argument is always bigger
*/

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main()
{
  int lower, upper, num;

  srand(time(0));
  printf("Enter lower limit : ");
  scanf("%d", &lower);
  printf("Enter upper limit : ");
  scanf("%d", &upper);

  num = (rand() % (upper - lower + 1)) + lower;
  printf("\nThe random number is : %d\n", num);

  return 0;
}
